docs/source/markdown/podman-auto-update.1.md

```diff
@@ -58,6 +58,11 @@ $ podman generate systemd --new --files bc219740a210455fa27deacc96d50a9e20516492
 # Load the new systemd unit and start it
 $ mv ./container-bc219740a210455fa27deacc96d50a9e20516492f1417507c13ce1533dbdcd9d.service ~/.config/systemd/user
 $ systemctl --user daemon-reload
+
+# The corresponding pod/container previously created must be removed before starting the pod/container through the systemd service
+$ podman stop bc219740a210455fa27deacc96d50a9e20516492f1417507c13ce1533dbdcd9d
+$podman rm bc219740a210455fa27deacc96d50a9e20516492f1417507c13ce1533dbdcd9d
```

Comment:
This one must not be replaced.  I just reran the script.

Reply:
I'll fix it after we figure out the complete instruction

Reply:
Fixed